lang_tests/run.rs

```diff
@@ -0,0 +1,34 @@
+use std::{fs::read_to_string, process::Command};
+
+use lang_tester::LangTester;
+
+static COMMENT_PREFIX: &str = "//";
+static LANG_TEST_PATH: &str = "lang_tests/tests";
+
+fn main() {
+    LangTester::new()
+        .test_dir(LANG_TEST_PATH)
+        .test_file_filter(|p| p.extension().unwrap().to_str().unwrap() == "yaiwr")
+        // Extract the first sequence of commented line(s) as the tests.
+        .test_extract(|p| {
+            read_to_string(p)
+                .unwrap()
+                .lines()
+                .skip_while(|l| !l.starts_with(COMMENT_PREFIX))
+                .take_while(|l| l.starts_with(COMMENT_PREFIX))
+                .map(|l| &l[COMMENT_PREFIX.len()..])
+                .collect::<Vec<_>>()
+                .join("\n")
+        })
+        .test_cmds(move |p| {
+            // Compile
+            let mut compiler = Command::new("cargo");
+            compiler.args(&["build", "--release"]);
+            // Run
+            let mut runtime = Command::new("./target/release/yaiwr");
+            runtime.args(&[p.to_str().unwrap()]);
+            vec![("Compiler", compiler), ("Run-time", runtime)]
```

Comment:
I actually don't need to compile anything here, but looks like tests_cmd expect me to provide a tuple with "Compiler" :thinking: 
If I commit it from result, I get:
```
Fatal exception:
  Command name(s) 'compiler' in tests are not found in the actual commands.
```

Reply:
It's because your tests have a `Compiler` line in them.

Reply:
That makes sense, thanks. Will remove.